tests/requirements.txt

```diff
@@ -0,0 +1,27 @@
+bok_choy==0.3.2
```

Comment:
@mtyaka I'm not sure bok_choy is actually used anywhere in the mentoring. Is it really required? Also, looks like it's just an output of `pip freeze`, which is not bad by itself, but I believe tests does not need acid-block and webob and pylint, etc. Could you please take a look at it and remove dependencies not actually used?